src/js/views/clinicians/sidebar/clinician-sidebar.hbs

```diff
@@ -6,5 +6,5 @@
 <div class="flex u-margin--t-8"><h4 class="sidebar__label u-margin--t-8">{{ @intl.clinicians.sidebar.clinicianSidebarTemplate.role }}</h4><div class="flex-grow" data-role-region></div></div>
 <div class="flex u-margin--t-8"><h4 class="sidebar__label u-margin--t-8">{{ @intl.clinicians.sidebar.clinicianSidebarTemplate.team }}</h4><div class="flex-grow" data-team-region></div></div>
 <div class="flex u-margin--t-8"><h4 class="sidebar__label u-margin--t-8">{{ @intl.clinicians.sidebar.clinicianSidebarTemplate.workspaces }}</h4><div class="flex-grow clinician-sidebar__workspaces" data-workspaces-region></div></div>
-<div class="flex u-margin--t-8"><div class="flex-grow" data-info-region></div></div>
+<div class="flex u-margin--t-8"><div class="sidebar__label u-margin--t-8"></div><div class="flex-grow" data-info-region></div></div>
```

Comment:
Added this element so the spacing would match the other sections. But did a `<div>` instead of a `<h4>`.